android/forms-screenlet-demo/app/src/main/java/com/liferay/mobile/formsscreenletdemo/util/DemoUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.liferay.mobile.formsscreenletdemo.util;
+
+import android.content.Context;
+import android.support.v4.content.ContextCompat;
+import android.view.View;
+import android.view.Window;
+import com.liferay.mobile.formsscreenletdemo.R;
+import com.liferay.mobile.screens.context.SessionContext;
+
+/**
+ * @author Luísa Lima
+ * @author Victor Oliveira
+ */
+public class DemoUtil {
+
+	public static String getCredentials() {
+		try {
+			return SessionContext.getCredentialsFromCurrentSession();
+		} catch (Exception e) {
+			return "";
+		}
+	}
+
+	public static String getResourcePath(String serverURL, long id, ResourceType resourceType) {
```

Comment:
Ao invés desse método receber um Enum, pq n receber já o valor da constante? Quem chamar fica responsável por chamar o correto. @phcp @camisLuisa

Reply:
Okay, that makes sense. I will do this after refactoring the code with the clean architecture.